raiden/utils/cli.py

```diff
@@ -201,6 +202,27 @@ def convert(self, value, param, ctx):
             self.fail('Please specify a valid hex-encoded address.')
 
 
+class LogLevelConfigType(click.ParamType):
+    name = 'log-config'
+    _validate_re = re.compile(
+        r'^(?:'
+        r'(?P<logger_name>[a-zA-Z0-9._]+)?'
+        r':'
+        r'(?P<logger_level>debug|info|warn(?:ing)?|error|critical|fatal)'
+        r',?)+$',
```

Comment:
didnt we use ; previously?

Reply:
Don't remember. What was `;` doing?

Reply:
aahh he was talking about the separator ...

Okay if so then yes it was comma before.